SETUP/db_schema.sql

```diff
@@ -522,6 +522,7 @@ CREATE TABLE `special_days` (
   `date_changes` varchar(100) default NULL,
   `info_url` varchar(255) default NULL,
   `image_url` varchar(255) default NULL,
+  `symbol` varchar(2) default '',
```

Comment:
I'm not going to pretend to know how mysql handles unicode but if it uses utf-16 like javascript with surrogate pairs and compex emoji you may need more space.

Reply:
In MySQL 5.5 and later, a single character is a single Unicode character regardless of how wide it is. So `varchar(2)` will allow any two Unicode characters.